File: Programmers/210508_KAKAO인턴/1.py
def solution(s):
    answer = ''
    info = ['zero', 'one', 'two', 'three', 'four', 'five', 'six', 'seven', 'eight', 'nine']
    info_two = '0123456789'
    s_list = []
    for i in s:
        s_list.append(i)

    cnt = 0

    string = ''
    while s_list:
        string += s_list.pop(0)
        if string in info_two:
            answer += string
            string = ''
        else:
            if string in info:
                answer += str(info.index(string))
                string = ''

    return int(answer)

File: 기출/2021kakao/1.js
function solution(gift_cards, wants) {
  let answer = 0;
  let cnt = 0
  let info = [];



  for (let i = 0; i < wants.length; i++) {
    info.push(0)
  }
  
  for (let i = 0; i < gift_cards.length; i++) {
    for (let j = 0; j < wants.length; j++) {
      if (gift_cards[i] === wants[j] && info[j] === 0) {
        info[j] = 1
        cnt += 1
        break
      }
    }
  }

  answer = wants.length - cnt
  // console.log(answer);
  return answer;
}

solution([5, 4, 5, 4, 5], [1, 2, 3, 5, 4])